bats/tests/preferences/move-from-roaming-to-local.bats

```diff
@@ -0,0 +1,36 @@
+load '../helpers/load'
+
+local_setup() {
+    if ! is_windows; then
+        skip 'appdata => localData migration is windows-only'
+    fi
+}
+
+@test 'factory reset' {
+    factory_reset
+}
+
+@test 'start app, create a setting, and move settings to roaming' {
+    start_container_engine
+    wait_for_container_engine
+    rdctl api -X PUT --body '{ "version": '"$(get_setting .version)"', "WSL": {"integrations": { "beaker" : true }}}'
+    rdctl shutdown
+    roamingHome="$(win32env APPDATA)/rancher-desktop"
```

Comment:
We don't really do camelCase in bash; this should be `roaming_home` or `ROAMING_HOME`.

Reply:
done